index.bs

```diff
@@ -10518,126 +10522,140 @@ The characteristics of this property are as follows:
         declared with the [{{Unforgeable}}] extended attribute and <emu-val>true</emu-val> otherwise;
-    *   |G| is the [=attribute getter=], defined below; and
-    *   |S| is the [=attribute setter=], also defined below.
-*   <div algorithm>
-
-        The <dfn id="dfn-attribute-getter" export>attribute getter</dfn> is a <emu-val>Function</emu-val>
-        object whose behavior when invoked is as follows:
-
-        1.  Let |idlValue| be an IDL value determined as follows.
-        1.  If the [=attribute=] is a [=regular attribute=], then:
-            1.  Let |I| be the [=interface=]
-                whose [=interface prototype object=]
-                this property corresponding to the attribute appears on.
-
-                Note: This means that even if an implements statement was used to make
-                an attribute available on the interface, |I| is the interface
-                on the left hand side of the implements statement, and not the one
-                that the attribute was originally declared on.
-
-            1.  Let |O| be the <emu-val>this</emu-val> value.
-            1.  If |O| is a [=platform object=],
-                then [=perform a security check=], passing:
-                *   the platform object |O|,
-                *   the [=identifier=] of the [=attribute=], and
-                *   the type “getter”.
-            1.  If |O| is not a [=platform object=] that implements |I|, then:
-                1.  If the [=attribute=] was specified with the
-                    [{{LenientThis}}] [=extended attribute=],
-                    then return <emu-val>undefined</emu-val>.
-                1.  Otherwise, [=ECMAScript/throw=] a <emu-val>TypeError</emu-val>.
-            1.  Set |idlValue| to be the result of performing the actions listed in the description of the attribute that occur when getting
-                (or those listed in the description of the inherited attribute, if this attribute is declared to
-                [=inherit its getter=]),
-                with |O| as the object.
-        1.  Otherwise, the attribute is a [=static attribute=].
-            Set |idlValue| to be the result of performing the actions listed in the description of the attribute that occur when getting.
-        1.  Let |V| be the result of [=converted to an ECMAScript value|converting=]
-            |idlValue| to an ECMAScript value.
-        1.  Return |V|.
-    </div>
+    *   |G| is the [=attribute getter=] created given the attribute, the interface (or the interface
+        it's being mixed in to, if the interface is actually a mixin), and the [=relevant Realm=] of
+        the object that is the location of the property; and
+    *   |S| is the [=attribute setter=] created given the attribute, the interface (or the interface
+        it's being mixed in to, if the interface is actually a mixin), and the [=relevant Realm=] of
+        the object that is the location of the property.
```

Comment:
I'll go with not changing it for now as this area is related to the mixin mess and will get straightened out when that is.